Add pipeline support

solid03 opened the pull request.

solid03 committed:
Add pipeline support

Sebazzz commented on PSMenu/Public/Show-Menu.ps1:
```diff
@@ -74,117 +74,125 @@ https://github.com/Sebazzz/PSMenu
+        $MenuItems.ForEach({ $allMenuItems.Add($_) })
+    }
+    end {
+        Test-HostSupported
```
Shouldn’t this be in `Begin { … }`?

solid03 replied:
That makes sense!

Sebazzz commented on PSMenu/Public/Show-Menu.ps1:
```diff
@@ -74,117 +74,125 @@ https://github.com/Sebazzz/PSMenu
-    Test-HostSupported
-    Test-MenuItemArray -MenuItems $MenuItems
+    begin {
+        $allMenuItems = [System.Collections.Generic.List[System.Object]]::new()
```
Please rename to `$AllMenuItems`

Sebazzz reviewed (changes requested):
Could you look into the various reformattings that have been done? These make the diff very cluttered and hard to review. Thanks!

solid03 committed:
Add requested changes to pull request:

- Replaced white space to make diff easier to read.  Sorry about that 😅.
- Moved Test-HostSupported into the begin block
- Changed new variable to PascalCase.
